styles.css

```diff
@@ -121,27 +111,241 @@ h1 {
 }
 
 .contact-icon {
-  filter: opacity(0.7);
-  margin: 15px;
-  width: 30px;
+  color: #505f79;
+  margin: 25px;
+  size: 40px;
 }
 
 .contact-icon:first-child {
   margin-left: 0;
 }
 
-.social p {
+.social h3 {
   color: #7f8cff;
   font-size: 16px;
   font-weight: 500;
   line-height: 24px;
-  margin-top: 10px;
+  margin: 0 10px;
+  text-transform: uppercase;
+}
+
+/* Work section */
+
+.work {
+  display: grid;
+  padding: 25px;
+}
+
+.project {
+  background-color: #fff;
+  border-radius: 20px;
+  box-shadow: 0 3px 4px 0 #344563;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 24px 0;
+  padding: 24px;
+  width: 100%;
+}
+
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 Please fulfill the request made by the [previous reviewer](https://github.com/hienphan0111/Hien-Portfolio/pull/3#discussion_r1038080774), according to which you still have `no space` on the `right` side of the `work` section elements and in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=hdw5gHzcXazz2Jky-0) every work element is `centered` and given the `same` space value from all sides, please fulfill this request 🙏🙏🙏

![image](https://user-images.githubusercontent.com/63915024/205324666-42eaffaa-4ef7-4f65-8a11-b059acbe1dd8.png)